src/functionalTest/resources/features/F-204 - Get NoC Questions/common/204_case_creation_token_creation.td.json

```diff
@@ -0,0 +1,29 @@
+{
+  "_guid_" : "Prerequisite_Case_Creation_Call_for_Case_Assignment_Token_Creation",
+  "_extends_" : "Case_Creation_Token_Creation",
+
+  "user": {
+    "username": "befta.master.solicitor.becky@gmail.com",
+    "password": "[[$CCD_BEFTA_MASTER_SOLICITOR_1_PWD]]"
+  },
```

Comment:
This token is used by: `204_case_creation.td.json/Prerequisite_Case_Creation_Call_for_Case_Assignment` so users should probably match:  i.e. change to Richard.